react/src/components/Weather/CurrentWeather.js

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from 'react';
+import CurrentStyle from './CurrentW.module.css';
+import SearchStyle from './Search.module.css';
+import sun from '../../icons/desktop/weathersun.png';
+import rain from '../../icons/desktop/weatherrain.png';
+import cloud from '../../icons/desktop/weathercloud.png';
+import { weatherApi } from './Api';
+// import Search from './Search';
+
+
+function CurrentWeather() {
+
+    const [query, setQuery] = useState('');
+    const [weather, setWeather] = useState({});
+
+    // const handleOnSearchChange = (searchData) => {
+    //     const [lat, lon] = searchData.value.split(" ");
+
+    //     const searching = fetch(
+    //         fetch(`${weatherApi.base}weather?q=${query}&units=metric&APPID=${weatherApi.key}`)
+    //             .then(res => res.json())
+    //     )
```

Comment:
Unikamy commitowania kodu, który jest zakomentowany. O ile w trakcie pracy nad zadaniem, takie coś jest akceptowalne, to finalnie nie powinno znaleźć się w PR. Jeśli jest wyjątek, potrzebne jest wytłumaczenie dla czego ten kod jest zakomentowany, oraz kiedy go usunąć i/lub odkomentować.

Rozwiązania częściowe powinny być robione na dedykowanych gałęziach.